### PR TITLE
[muxcable] Remove Xcvrd Sleep

### PR DESCRIPTION
**Pull request by tahmed-dev**
#### Description

When using redis select, xcvrd is sleeping for 100msec which appears
unnecessary after process all content of selectable objects.

signed-off-by: Tamer Ahmed <tamer.ahmed@microsoft.com>

#### Motivation and Context
Reduce overall time it takes to switch multiple/all ports

#### How Has This Been Tested?
Test on test cluster where 24 ports were switched and saw 100msec to 200msec gain in switching time of all ports. Individual port switch time is still within 20msec~30msec.

### CODE REVIEW

**Review by jleveque — CHANGES_REQUESTED** [review]
Please fix LGTM alert (no longer `import time`)

**Comment by jleveque** [comment]
Has the sonic-swss-common submodule been updated to include the threading fix? If not, this PR should wait until then.

**Comment by tahmed-dev** [comment]
> Has the sonic-swss-common submodule been updated to include the threading fix? If not, this PR should wait until then.

test without `swss-common` threading fix and saw the gain mentioned in the description of this PR.

**Comment by vdahiya12** [comment]
last i checked main xcvrd thread stopped running as I could not see TRANSCEIVER_INFO table getting any updates, if i removed the sleep.

**Comment by tahmed-dev** [comment]
> last i checked main xcvrd thread stopped running as I could not see TRANSCEIVER_INFO table getting any updates, if i removed the sleep.

can you please try with the recent changes?

**Comment by jleveque** [comment]
I think we should wait for the sonic-swss-common submodule update. The threading issue is properly fixed upstream.

**Review by vdahiya12 — APPROVED** [review]

**Comment by tahmed-dev** [comment]
> Has the sonic-swss-common submodule been updated to include the threading fix? If not, this PR should wait until then.

@jleveque can you please unblock this PR?

**Review by jleveque — APPROVED** [review]

**Comment by tahmed-dev** [comment]
> Please fix LGTM alert (no longer `import time`)

@jleveque  There are requested changes that are blocking the merge of this PR.

**Pull request merged**
